pkg/lib/cockpit-components-table.jsx

```diff
@@ -259,9 +259,9 @@ export const ListingTable = ({
             <Table {...extraProps} {...tableProps}>
                 {showHeader && <Thead>
                     <Tr>
-                        {isExpandable && <Th />}
-                        {!onHeaderSelect && onSelect && <Th />}
-                        {onHeaderSelect && onSelect && <Th select={{
+                        {isExpandable && <Th aria-label="expander"/>}
+                        {!onHeaderSelect && onSelect && <Th aria-label="select"/>}
+                        {onHeaderSelect && onSelect && <Th aria-label="select" select={{
```

Comment:
These need to be translated. But moreover, should that really be a description of the widget type? Our existing code usually has specific descriptions of the thing, like "Add DNS server" or "Current top CPU usage".

Reply:
Nevermind, this is self-inflicted pain -- we literally create empty `<Th>`s here to fit the expander and/or select widgets. So a translatable version of the above seems fine indeed.